docs/baseline.md

```diff
@@ -0,0 +1,177 @@
+In other words, Baseline status aims to identify features of developers’ contemporary, workaday web.
+
+Goal 1: **Tailor Baseline status to a developer audience.**
+The Baseline status is intended for developers creating and maintaining hypertext applications for the web, who may not have access to the resources, data, tools, or knowledge needed to identify their users’ browsers capabilities more directly.
+See [Audience illustration for Baseline status](#audience-illustration-for-baseline-status) for a detailed story to help understand this goal in more detail.
+
+Goal 2: **Change with the web and web developers’ needs.** Features can and should join and leave Baseline status as their respective interoperability, reliability, commitment, and availability profiles change. Likewise, Baseline statuses should change as developers’ needs change (for example, developers come to rely on a new browser or an established browser fades from use). Baseline is not an inert, one-time assessment.
+
+Goal 3: **Identify interoperability through support across browser implementations.**
+Baseline features demonstrate consensus through multiple implementations.
+Baseline is not for single-implementation browser features.
+
+Goal 4: **Identify availability through popular browser support.**
+Baseline features demonstrate consensus through user reach, a supermajority of global users by reasonable measures used by web developers.
+Baseline features should not surprise a web developer with compatibility failures.
+
+Goal 5: **Identify continuity through support over time.**
+Baseline features demonstrate consensus through constancy: uninterrupted support from one release to the next.
+Baseline is not for prototypes, betas, or temporary hacks.
+All software has bugs, but Baseline features have narrow, known bugs.
+
+Goal 6: **Identify commitment through documentation.**
+Baseline features demonstrate consensus through specifications, documentation for web developers, and other signs that the feature will continue to have a place in a web developer’s toolbox.
+Baseline features aren’t unspecified or formally discouraged.
+
+<!-- TODO: https://github.com/web-platform-dx/web-features/issues/437 -->
+
+## Non-goals
+
+Baseline status cannot or will not satisfy the following non-goals:
+
+* **Identify universally available features.**
+  Many Baseline features will not achieve 100% user reach soon or perhaps ever.
+  If a web developer needs to support a globally uncommon or discontinued browser (e.g., Internet Explorer 11), then the developer needs to know the specific limitations of that browser, not a broad overview of developers’ most commonly required browsers.
+  Baseline can’t be both.
+* **Identify support in non-web environments.**
+  Developers use web technologies in non-web settings, such as JavaScript in Node.js, Web APIs in Deno, or HTML and CSS in Electron-based applications.
```

Comment:
Ah, this is a good thought though I don't know exactly where it sits either. I think it's more of a future consideration than a non-goal, but it's possible it could _become_ a non-goal. I filed https://github.com/web-platform-dx/web-features/issues/477 to take up this question.